test/recipes/aws-parallelcluster-install/controls/users.rb

```diff
@@ -0,0 +1,22 @@
+user = "pcluster-admin"
+
+control 'admin_user' do
+  title 'Configure cluster admin user'
+
+  describe command("grep #{user} /etc/passwd") do
+    its('stdout.strip') { should eq "#{user}:x:400:400:AWS ParallelCluster Admin user:/home/#{user}:/bin/bash"}
```

Comment:
Why did you decide to use this instead of leveraging `user` resource for all the checks?

Reply:
Because, strangely, with the user inSpec resource I cannot check the description https://docs.chef.io/inspec/resources/user/